packages/myst-cli/src/build/html/index.ts

```diff
@@ -154,10 +154,15 @@ export async function buildHtml(session: ISession, opts: StartOptions) {
   fs.copySync(session.publicPath(), path.join(htmlDir, 'build'));
   fs.copySync(path.join(session.sitePath(), 'config.json'), path.join(htmlDir, 'config.json'));
   fs.copySync(path.join(session.sitePath(), 'objects.inv'), path.join(htmlDir, 'objects.inv'));
-  fs.copySync(
-    path.join(session.sitePath(), 'myst.xref.json'),
-    path.join(htmlDir, 'myst.xref.json'),
-  );
+
+  // NOTE: HTML static output needs to patch the contents, this is done on the fly by the server
+  const xrefs = JSON.parse(
+    fs.readFileSync(path.join(session.sitePath(), 'myst.xref.json')).toString(),
+  ) as any;
```

Comment:
This should have an actual type (will need the import too `import type { MystXRefs } from 'myst-transforms'`);


```suggestion
  ) as MystXRefs;
```